app/src/main/java/com/duckduckgo/app/browser/BrowserTabViewModel.kt

```diff
@@ -941,93 +934,31 @@ class BrowserTabViewModel(
 
     fun onUserClickCtaOkButton(cta: Cta) {
         ctaViewModel.onUserClickCtaOkButton(cta)
-        viewModelScope.launch {
-            withContext(dispatchers.io()) {
-                ctaViewModel.obtainNextCta(previousCta = cta)
-            }?.let { ctaViewState.value = currentCtaViewState().copy(cta = it) }
-            produceNewCommand(cta)
-        }
-    }
-
-    fun onUserClickCtaSecondaryButton(cta: SecondaryButtonCta) {
-        ctaViewModel.onUserClickCtaSecondaryButton(cta)
-    }
-
-    fun onUserDismissedCta(dismissedCta: Cta) {
-        ctaViewModel.onUserDismissedCta(dismissedCta)
-        when (dismissedCta) {
-            is HomeTopPanelCta -> {
-                if (!variantManager.getVariant().hasFeature(VariantManager.VariantFeature.ConceptTest)) {
-                    refreshCta()
-                } else {
-                    ctaViewState.value = currentCtaViewState().copy(cta = null)
-                }
-            }
-            is HomePanelCta -> refreshCta()
-            else -> ctaViewState.value = currentCtaViewState().copy(cta = null)
-        }
-    }
-
-    private fun produceNewCommand(cta: Cta) {
         command.value = when (cta) {
             is HomePanelCta.Survey -> LaunchSurvey(cta.survey)
             is HomePanelCta.AddWidgetAuto -> LaunchAddWidget
             is HomePanelCta.AddWidgetInstructions -> LaunchLegacyAddWidget
-            is DaxDialogCta.DefaultBrowserCta -> cta.primaryAction.mapToCommand()
-            is DaxDialogCta.SearchWidgetCta -> cta.primaryAction.mapToCommand()
             else -> return
         }
     }
 
-    fun onUserTriedToSetAsDefaultBrowserFromSettings() {
-        val isDefaultBrowser = defaultBrowserDetector.isDefaultBrowser()
-        installStore.defaultBrowser = isDefaultBrowser
-        firePixelDefaultBrowserCtaUserAction(isDefaultBrowser, origin = Pixel.PixelValues.DEFAULT_BROWSER_SETTINGS)
+    fun onUserClickCtaSecondaryButton(cta: SecondaryButtonCta) {
```

Comment:
That removal was avoided on purpose because of the privacy coach project and others that could make use of this very soon. But with the latest updates, we might want to remove it too.  What do you think? should we remove it?

Reply:
I talked to product and we could use it, so let's keep it and if it doesn't get used we'll remove it as a LHF task.

Reply:
Tracked internally: https://app.asana.com/0/1149059203486286/1169079212142450/f